src/lib/netlink/ipoib.rs

```diff
@@ -0,0 +1,4 @@
+use crate::IpoibInfo;
+use netlink_packet_route::rtnl::link::nlas::InfoIpoib;
+
+pub(crate) fn parse_ipoib_info(infos: &[Info]
```

Comment:
This file is not finished?

Reply:
I've removed. That was initial work not needed any more. `parse_ipoib_info` is now done in `src/lib/ifaces/ipoib.rs` as we are using `InfoIpoib` from `netlink_packet_route` and netlink parsing is not needed.

Everything is updated now. Sorry for this.